### PR TITLE
fix(navbar): fixed the position of the navabr

### PR DESCRIPTION
**Pull request by smilegupta**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -46,4 +49,4 @@ function App() {
     );
 }
 
-export default App;
+export default withWrapper(App);
```
Comment (rparsh) [inline src/App.js:52]: do we need withWrapper ?

Comment (InvincibleZeal) [reply src/App.js:52]: Not really, since we implemented the spacing HOC for learning purposes. So we are stilling using it.

**Comment by gautism1** [comment]
now i18 button is working as it should be

**Comment by gautism1** [comment]
if we have some wrong url like. https://deploy-preview-84--billbook-razorpay.netlify.app/customers/addss
then it is routing to /Customer,
we can some Page not Found type something ?
<Route component={PageNotFound} />

**Review by InvincibleZeal — APPROVED** [review]

**Pull request merged**